app/src/main/java/openfoodfacts/github/scrachx/openfood/views/LoginActivity.java

```diff
@@ -147,7 +145,7 @@ public void onResponse(Call<ResponseBody> call, Response<ResponseBody> response)
                 } else {
                     // store the user session id (user_session and user_id)
                     for (HttpCookie httpCookie : HttpCookie.parse(response.headers().get("set-cookie"))) {
-                        if (httpCookie.getDomain().equals(".openfoodfacts.org") && httpCookie.getPath().equals("/")) {
+                        if (httpCookie.getDomain().equals(".openbeautyfacts.org") && httpCookie.getPath().equals("/")) {
```

Comment:
shouldn't this domain string be based on the flavor instead?

Reply:
But the Domain returned after login is this .openbeautyfacts.org

Reply:
@policorn sry i didn't get what you said. I printed the log and find that Domain name is .openbeautyfacts.org not the .openfoodfacts.org so comparing httpCookie.getDomain() with .openbeautyfacts.org will set the condition true and we will store the user and session and it will resolve the issue #996

Reply:
I think what he is trying to say is that depending on the current flavour of the app, the domain should change.

If it is the OFF flavor, use `.openfoodfacts.org`
If it is the OBF flavor use `.openbeautfyfacts.org`

Reply:
.openpetfoodfacts.org for OPFF

Reply:
Thanks @Karljoones, that's right; sorry @danzzen I wasn't more clear

Reply:
No problem @policorn